src/components/icons.tsx

```diff
@@ -0,0 +1,25 @@
+import { Spinner, SpinnerProps } from '@terra-ui-packages/components';
```

Comment:
Git didn't recognize this as a rename due to the high percentage of lines that changed.